.github/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,49 @@
+# Issue triage
+
+This document describes the triage process used for the [odata.net repository].
+
+## Prioritization
+
+We use customer feedback as our primary input for prioritizing issues.
+
+* P1: 5 or more asks, legal or security issues
+* P2: 3-4 asks, regressions
+* P3: 2 asks
+* P4: 1 ask
+
+## What constitutes an ask
+
+We do our best to interpret customer feedback as asks. The easiest thing for us to interpret is opening an issue or an explicit comment on the issue stating that the fix would be helpful to a different party. We also make a best effort to interpret positive reactions as asks, although this is more subjective.
+
+Duplicate issues also constitute an ask. You can help us identify duplicate issues by adding a comment to the issue that references the duplicate issue number.
+
+Finally, we also count asks from partner teams at Microsoft. We strongly encourage our colleagues to post the ask directly on GitHub, however that doesn't always happen.

```

Comment:
super nit: "however" is not a conjunction so we can write that part as "...GitHub; however..." or "...GitHub, but..."